engine/common/synchronization/engine.go

```diff
@@ -519,11 +583,14 @@ func (e *Engine) validateBlockResponseForALSP(channel channels.Channel, id flow.
 	return nil, true, nil
 }
 
-// validateRangeRequestForALSP checks if a range request should be reported as a misbehavior.
+// validateRangeRequestForALSP checks if a range request should be reported as a misbehavior due to malicious intent (e.g. spamming).
```

Comment:
can you update the godocs on the functions in this file to match the standard structure like the comment above.

Reply:
6999004a1f43e6a6376442b8f4b28b55bb01bcb0